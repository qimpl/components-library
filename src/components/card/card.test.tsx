import React from 'react';

import { render } from '@testing-library/react';

import Text, { TextVariant } from 'components/text';
import Title, { TitleVariant } from 'components/title';

import Card from '.';

test('Default card', () => {
  expect(
    render(
      <Card>
        <>
          <Title variant={TitleVariant.H1}>Custom List Item</Title>
          <Text variant={TextVariant.MEDIUM}>List Item 1</Text>
          <Text variant={TextVariant.MEDIUM}>List Item 2</Text>
          <Text variant={TextVariant.MEDIUM}>List Item 3</Text>
        </>
      </Card>
    )
  ).toMatchSnapshot();
});
