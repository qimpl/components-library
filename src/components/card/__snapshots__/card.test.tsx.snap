// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default card 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-gKAaRy jbNfMh"
      >
        <h1
          class="sc-dlnjwi gYKPDG"
          font-size="20px"
        >
          Custom List Item
        </h1>
        <p
          class="sc-bdnxRM icgxZr"
          style="color: rgb(1, 60, 99); font-size: 15px; line-height: 21px;"
        >
          List Item 1
        </p>
        <p
          class="sc-bdnxRM icgxZr"
          style="color: rgb(1, 60, 99); font-size: 15px; line-height: 21px;"
        >
          List Item 2
        </p>
        <p
          class="sc-bdnxRM icgxZr"
          style="color: rgb(1, 60, 99); font-size: 15px; line-height: 21px;"
        >
          List Item 3
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-gKAaRy jbNfMh"
    >
      <h1
        class="sc-dlnjwi gYKPDG"
        font-size="20px"
      >
        Custom List Item
      </h1>
      <p
        class="sc-bdnxRM icgxZr"
        style="color: rgb(1, 60, 99); font-size: 15px; line-height: 21px;"
      >
        List Item 1
      </p>
      <p
        class="sc-bdnxRM icgxZr"
        style="color: rgb(1, 60, 99); font-size: 15px; line-height: 21px;"
      >
        List Item 2
      </p>
      <p
        class="sc-bdnxRM icgxZr"
        style="color: rgb(1, 60, 99); font-size: 15px; line-height: 21px;"
      >
        List Item 3
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
